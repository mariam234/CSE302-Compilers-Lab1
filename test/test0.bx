x = 3;
y = (x + 2) * (x - 2);
z = - y;
print z;

b = 5;
u = b * b;
a = 1;
c = 2;
v = 4 * a * c;
w = u - v;
print w;

x = 1;
y = 5;
z = 3;
z = x + y;
print z;
