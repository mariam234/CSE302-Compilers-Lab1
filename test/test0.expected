5
-17
6
-3
0
1
3
-5
480
18691697672192
-2
0
-91203129
16
-4356
